include/xsimd/arch/xsimd_sse4_1.hpp

```diff
@@ -133,15 +133,15 @@ namespace xsimd {
     batch<T, A> select(batch_bool_constant<batch<T, A>, Values...> const&, batch<T, A> const& true_br, batch<T, A> const& false_br, requires_arch<sse4_1>) {
       constexpr int mask = batch_bool_constant<batch<T, A>, Values...>::mask();
       switch(sizeof(T)) {
-        case 2: return _mm_blend_epi16(false_br, true_br, mask);
+        case 2: return _mm_blend_epi16(false_br, true_br, mask & 0xFF);
```

Comment:
https://ci.appveyor.com/project/xtensor-stack/xsimd/builds/40931948/job/jhyfbjdd791gxd0n#L652
Thought these warnings don't make much sense IMO.

Reply:
Turns out latest msvc 2019 has fixed this issue.